src/Context/LikeContext/LikeContext.js

```diff
@@ -0,0 +1,55 @@
+import axios from "axios";
+import { createContext, useContext, useEffect, useState } from "react";
+import { getLikedVideos } from "../../Services/likeService";
+import { useAuth } from "../Auth/auth-context";
+const LikeContext = createContext();
+const LikeProvider = ({ children }) => {
+  const [likedVideos, setLikedVideos] = useState([]);
+  const { auth } = useAuth();
+  useEffect(() => {
+    if (auth.isLogIn) {
+      (async () => {
+        try {
+          const response = await getLikedVideos(auth.tokenValue);
+          setLikedVideos(response.data.likes);
+        } catch (error) {
+          console.log("error occured while getting liked videos", error);
```

Comment:
suggestion: for errors you can use `console.error` method

Reply:
Thanks for suggesting that